rules/import_middleman.bzl

```diff
@@ -210,8 +210,18 @@ def _file_collector_rule_impl(ctx):
     existing_static_framework = objc_provider_fields.get("static_framework_file", depset([]))
 
     deduped_static_framework = depset(_deduplicate_test_deps(test_linker_deps[0], existing_static_framework.to_list()))
+
     replaced_static_framework = _replace_inputs(ctx, deduped_static_framework, input_static_frameworks, _update_framework)
-    objc_provider_fields["static_framework_file"] = depset(replaced_static_framework.inputs)
+    rules_apple_api_version = getattr(bundling_support, "rule_api_version", None)
+    use_lts_5_rules_apple_api = rules_apple_api_version == 1.0
+    if use_lts_5_rules_apple_api:
+        objc_provider_fields["static_framework_file"] = depset(replaced_static_framework.inputs)
+    else:
+        # See comment in file above regarding this name in 6.x.x
```

Comment:
Is this referring to L103? It might be more straightforward to repeat it here

Reply:
Yes, I have added a `Notes` segment to the top of the doc WRT this: https://github.com/bazel-ios/rules_ios/pull/767/files#diff-e213489d3a65f173ffa46728826b06f786a5d63666e3fe7101493397ee12f948R2